rules-reviewed/eap6/java-ee/java-ee-jaxrpc.windup.xml

```diff
@@ -22,14 +22,22 @@
                     <location>INHERITANCE</location>
                 </javaclass>
             </when>
-            <perform>
-                <classification title="JAX-RPC Generic Handler" effort="1" severity="mandatory">
-                    <link href="http://docs.oracle.com/cd/E17802_01/webservices/webservices/docs/1.6/api/javax/xml/rpc/handler/GenericHandler.html"
-                        title="Java EE RPC Generic Handler API" />
-                    <tag>jax-rpc</tag>
-                    <tag>soap</tag>
-                </classification>
-            </perform>
-        </rule>
+              <perform>
+                  <hint title="JAX-RPC GenericHandler usage" effort="0" severity="optional">
+                      <message>
+                        <![CDATA[
+                        The "javax.xml.rpc.handler.GenericHandler" class implements the Handler interface.
+                        SOAP Message Handler developers should typically subclass GenericHandler class unless
+                        the Handler class needs another class as a superclass. It is a convenience abstract
+                        class that makes writing Handlers easy. GenericHandler provides default implementations
+                        of the lifecycle methods init and destroy and also different handle methods.
+                        ]]>
```

Comment:
Why use CDATA if in the message element is not any XML dedicated character? It is not required ;)

btw what purpose are quotes in "javax.xml.rpc.handler.GenericHandler" ? Is it typo to use single quote => ` ?